target/linux/brcm47xx/image/Makefile

```diff
@@ -893,7 +1012,8 @@ endef
 #  TARGET_DEVICES += netgear-wndr3400-vcna
 
 define Device/netgear-wndr4000
-  DEVICE_TITLE := Netgear WNDR4000
+  DEVICE_VENDOR := Netgear
+  DEVICE_MODEL := WNDR4000
```

Comment:
Maybe add v1 here, since all of the other Netgear devices have revisions, too.

Reply:
done